apps/web/pages/api/auth/forgot-password.ts

```diff
@@ -33,7 +33,7 @@ async function handler(req: NextApiRequest, res: NextApiResponse) {
   });
 
   try {
-    const user = await prisma.user.findUnique({
+    const user = await prisma.user.findFirst({
```

Comment:
Finding the first user with an email to apply forget password is fine because then when setting the email at the end of the process it will be applied to every profile using the same email. Essentially every user row in Users table with the same email will have the same password.